documentation/Install_PACTA_as_a_first-time_user

```diff
@@ -0,0 +1,137 @@
+
+### Step 2 — Launch RStudio:
+To launch RStudio with the PACTA_analysis files:
+* From your local directory open` PACTA_analysis > R > PACTA_analysis.rproj `
+
+### Step 3 — Install the required packages
+Install all the required packages listed in Getting Started.
+
+### Step 4 — Running web_tool_script1 and web_tool_script2:
+The scripts use the following example files located in `working_directory`:
+* `20_Raw_Inputs > TestPortolio_Input.csv`
+* `10_Parameter_File > TestPortfolio_Input_PortfolioParameters.yml`
+	* The `portfolio_name_in` variable must match the `.csv` file name
+
+#### To run web_tool_script_1.r:
+1. Open `web_tool_script_1.r` in RStudio
+2. Click Source to run the script
+3. To confirm the script was successful, navigate to: `working_directory > 30_Processed_Inputs > TestPortfolio_Input`. The generated folder should contain new data and objects
+4. Inspect these two files before moving on:
+	* `audit_file.csv`
+	* `invalidsecurities.csv`
+
+> These files are your first indication if something has gone wrong with a portfolio. They identify what data is valid or invalid for the analysis.
+
+#### To run web_tool_script_2.r:
+1. Open `web_tool_script_2.r` in RStudio
+2. Click Source to run the script.
+3. To confirm that the script was successful, navigate to: `working directory > 40_Results > TestPortfolio_Inputs`. It should contain the following 6 files:
+	* `Bonds_results_company.rda`
+	* `Bonds_results_map.rda`
+	* `Bonds_results_portfolio.rda`
+	* `Equity_results_company.rda`
+	* `Equity_results_map.rda`
+	* `Equity_results_portfolio.rda`
+
+> To open the `.rda` files: `R> readRDS(“”) `
+> ***This is a specific method to this application only `.rda` files are click to open in all other cases.***
+
+- - -
+# Next Steps:
+If the scripts were successful with the test data, you are ready to run the analysis using your own files and generate interactive reports.
+## Running an analysis with your own data:
+### Step 1 — Reset the environment:
+1. Restart R.
+2. Manually remove the generated `TestPortfolio_Input` folders from the `working_directory`. It is recommended to keep one copy for reference.
+
+### Step 2 — Load your portfolio files:
+1. Copy your working portfolio into `20_Raw_Inputs`
+2. Go to `10_Parameter_Files`. Make a copy of `Portfolio_Input_PortfolioParamaters.yml`. Rename the file to `YOURPORTFOLIONAME_PortfolioParameters.yml.`
+3. View the file in RStudio and edit the following variables:
+	* `portfolio_name_in`: your file name without `.csv`
+	* `investor_name_in:` anything
+	* `peergroup:` anything
+	* `language:`available languages?
+	* `project_code:` GENERAL
+	* — placeholder for upcoming feautre— `holdings_date:`
```

Comment:
Placeholder for holdings_date feature to be documented, will we include this in this draft?

Reply:
yes please.

`holdings_date: 2020Q4`